__tests__/articles.test.js

```diff
@@ -99,3 +99,57 @@ describe("/api/articles", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  describe("GET", () => {
+    describe("Status 200", () => {
+      it("should respond with an array of comments for the given article_id", () => {
+        return request(app)
+          .get("/api/articles/1/comments")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.comments).toBeInstanceOf(Array);
+            expect(body.comments[0]).toHaveProperty("comment_id");
+            expect(body.comments[0]).toHaveProperty("votes");
+            expect(body.comments[0]).toHaveProperty("created_at");
+            expect(body.comments[0]).toHaveProperty("author");
+            expect(body.comments[0]).toHaveProperty("body");
+            expect(body.comments[0]).toHaveProperty("article_id");
```

Comment:
It would be beneficial to be more specific here and check that the article id is the number you expect, for example in this test it would be 1. This makes sure we're getting the correct comments back!